subt/offseter.py

```diff
@@ -0,0 +1,104 @@
+from threading import Thread
+
+class Offseter:
```

Comment:
why don't you inherit from `osgar.node:Node`?

Reply:
Because it is not really helping me (you ask every time and the answer is the same every time).

Reply:
OTOH if it does not "match" `osgar.Node` it is causing other issues (A drone, 18437fc9-a228-4e6e-901c-0a3f7f5c5814):
```
python -m osgar.replay --module offseter logs\aws\Q\ver78qb\aws-ver78qb-A80RH170L80H200L.log
original args: b"['/osgar-ws/src/osgar/subt/__main__.py', 'run', '/osgar-ws/src/osgar/subt/zmq-subt-x4.json', '--log', '/osgar-ws/logs/zmq-subt-x4-2020-10-01T10.41.23.log', '--side', 'auto', '--walldist', '1.2', '--speed', '1.5', '--note', 'run_solution.bash']"
Traceback (most recent call last):
  File "C:\SDK\WinPython-64bit-3.6.2.0Qt5\python-3.6.2.amd64\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "C:\SDK\WinPython-64bit-3.6.2.0Qt5\python-3.6.2.amd64\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "D:\osgar\osgar\replay.py", line 111, in <module>
    main()
  File "D:\osgar\osgar\replay.py", line 94, in main
    module_instance = replay(args)
  File "D:\osgar\osgar\replay.py", line 62, in replay
    module_instance = module_class(module_config.get('init', {}), bus=bus)
  File "D:\osgar\subt\offseter.py", line 8, in __init__
    self.thread.name = bus.name
AttributeError: 'LogBusHandler' object has no attribute 'name'
```
so let's do something with `osgar.Node` (at least added it do "dept list") ... also I expected that then the switch between Threads/Processes would be smooth, but maybe I am mixing different things ...

Reply:
This has nothing to do with `osgar.Node`. The code here is expecting that `bus` has a `name`... But `osgar.Node` and all the `*Handlers` definitely should go to the debt list #640 (there shouldn't be 5 different handlers with the need to synchronize their API).

Switch between threads and processes is smooth already - either one works (if it doesn't then it is a bug) - so that is not related.